### DISCUSSION TITLE
isSubmitted set to true after first rerender after reset was called

### DISCUSSION THREAD

**Discussion by teofilp** [react-hook-form/react-hook-form#4581] [Q&A]
Hi, I was trying to show some errors on inputs only if the form was submitted and I tried using isSubmitted from the formState. I played a bit with this playground https://codesandbox.io/s/react-hook-form-reset-issubmitted-state-after-form-submitted-forked-smzy9?file=/src/index.js and I found something interesting going on. If you check the console, I output the value of the isSubmitted and after completing the form and pressing submit you can see in the console that isSubmitted is false, and then true again. Is that ok? Shouldn't isSubmitted remain false after the reset was done, before doing another submit?

**Comment by bluebill1049** [comment]
can you try the latest v6?

**Reply by teofilp** [reply]
This is with 6.15.3 (https://codesandbox.io/s/react-hook-form-reset-issubmitted-state-after-form-submitted-forked-smzy9?file=/src/index.js) and I see the same behaviour
